CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# dbt_marketo_source v0.8.0
+PR [#28](https://github.com/fivetran/dbt_marketo_source/pull/28) incorporates the following updates:
+## 🚨 Breaking Changes 🚨
+Some of the more complex transformation logic has been moved from the Marketo source package to the transform package. This was done so the delineation between staging and intermediate models is in line with Fivetran's other packages. This does not affect the final tables created by the transform package, but this will affect the staging tables as outlined below. 
+- Model `stg_marketo__lead_base` from `dbt_marketo_source` has been rolled into [`stg_marketo__lead`](https://github.com/fivetran/dbt_marketo_source/blob/main/models/stg_marketo__lead.sql).
+- Parts from model `stg_marketo__lead` from `dbt_marketo_source` have been moved to a new model [`int_marketo__lead`](https://github.com/fivetran/dbt_marketo/blob/feature/create-intermediates/models/intermediate/int_marketo__lead.sql) in `dbt_marketo`.
+- The default schema for the source tables are now built within a schema titled (`<target_schema>` + `_marketo_source`) in your destination. The previous default schema was (`<target_schema>` + `_stg_marketo`). This may be overwritten if desired.
+## Features
+- 🎉 Databricks compatibility 🎉
```

Comment:
Since you technically added Postgres compatibility in this update, you should probably highlight that here as well!

Reply:
Done!